net-irc/hexchat/hexchat-2.12.1.ebuild

```diff
@@ -0,0 +1,142 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+inherit fdo-mime gnome2-utils mono-env multilib python-single-r1
+
+DESCRIPTION="Graphical IRC client based on XChat"
+HOMEPAGE="https://hexchat.github.io/"
+
+if [[ "${PV}" == "9999" ]] ; then
+	inherit git-r3
+	SRC_URI=""
+	EGIT_REPO_URI="git://github.com/hexchat/hexchat.git"
+else
+	SRC_URI="https://dl.hexchat.net/hexchat/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux"
+fi
+
+LICENSE="GPL-2 plugin-fishlim? ( MIT )"
+SLOT="0"
+IUSE="dbus debug +gtk libcanberra libnotify libproxy libressl lua nls perl +plugins plugin-checksum plugin-doat plugin-fishlim plugin-sysinfo python spell ssl theme-manager"
+REQUIRED_USE="
+	python? ( plugins ${PYTHON_REQUIRED_USE} )
+	perl? ( plugins )
+	lua? ( plugins )
+
+	plugin-checksum? ( plugins )
+	plugin-doat? ( plugins )
+	plugin-fishlim? ( plugins )
+	plugin-sysinfo? ( plugins )
+"
+
+COMMON_DEPEND="dev-libs/glib:2
+	dbus? ( sys-apps/dbus )
+	gtk? ( x11-libs/gtk+:3 )
+	libcanberra? ( media-libs/libcanberra )
+	libproxy? ( net-libs/libproxy )
+	libnotify? ( x11-libs/libnotify )
+	lua? ( dev-lang/lua:= )
+	nls? ( virtual/libintl )
+	plugins? (
+		perl? ( dev-lang/perl )
+		plugin-sysinfo? ( sys-apps/pciutils )
+		python? ( ${PYTHON_DEPS} )
+	)
+	spell? ( app-text/iso-codes )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	theme-manager? ( dev-lang/mono )"
+RDEPEND="${COMMON_DEPEND}
+	spell? ( app-text/enchant )"
+DEPEND="${COMMON_DEPEND}
+	app-arch/xz-utils
+	virtual/pkgconfig
+	dev-util/intltool
+	theme-manager? ( dev-util/monodevelop )"
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+	if use theme-manager ; then
+		mono-env_pkg_setup
+		export XDG_CACHE_HOME="${T}/.cache"
+	fi
+}
+
+src_configure() {
+	econf \
+		$(use_enable nls) \
+		$(use_enable ssl openssl) \
+		$(use_enable gtk gtkfe) \
+		$(use_enable !gtk textfe) \
+		$(usex python "--enable-python=${EPYTHON}" "--disable-python") \
```

Comment:
That won't work here, we need `--enable-python=python2.7`, not `--enable-python2.7`